opflexagent/test/test_gbp_ovs_agent.py

```diff
@@ -155,7 +155,7 @@ def test_port_unbound_snat_cleanup(self):
         args_2['port'].gbp_details = mapping
```

Comment:
You may want to add:
        self.agent.ep_manager.snat_iptables.setup_snat_for_es.return_value = (
             tuple('foo-if', 'aa:bb:cc:00:11:44'))

before the next call to self.agent.try_port_binding(**args_2) on line 156. That way I think yu can keep the assertEqual to just 1.

Reply:
I have updated the tests accordingly. Please take a look.